CMakeLists.txt

```diff
@@ -9,30 +9,44 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 option(RAMTOOLS_BUILD_TESTS "Build unit tests" ON)
 option(RAMTOOLS_BUILD_BENCHMARKS "Build benchmarks" ON)
+option(RAMTOOLS_BUILD_TOOLS "Build tools" ON)
 
 find_package(ROOT 6.26 REQUIRED COMPONENTS Core RIO Tree ROOTNTuple ROOTNTupleUtil)
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc)
+include(${ROOT_USE_FILE})
 
-add_library(ramtools_tree STATIC
+include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc/ttree)
```

Comment:
We should probably use `target_include_directories`. `include_directories` is not recommended unless there is a good reason for it.